Makefile.riot

```diff
@@ -1,7 +1,9 @@
 MODULE = tinydtls
 
-CFLAGS += -DDTLSv12 -DWITH_SHA256
+SRC := ccm.c  crypto.c  dtls.c  dtls_time.c  hmac.c  netq.c  peer.c  session.c dtls_prng.c
 
-SRC := ccm.c  crypto.c  dtls.c  dtls_debug.c  dtls_time.c  hmac.c  netq.c  peer.c  session.c dtls_prng.c
+ifneq (,$(filter tinydtls_debug, $(USEMODULE)))
+  SRC += dtls_debug.c
```

Comment:
There are functions declared in `dtls_debug.h`, which are removed by this.

```
const char *dtls_package_name(void);
const char *dtls_package_version(void);
log_t dtls_get_log_level(void);
void dtls_set_log_level(log_t level);
void dtls_set_log_handler(dtls_log_handler_t app_handler);
```

I would consider to keep the `dtls_package_name` and `dtls_package_version`.  If agreed, then the `dtls_debug.c` should not be excluded.

The "log" functions may then be added to the empty inlines.
And the excluded section in "dtls_debig.c"  (NDEBUG) must be adjusted.